### DISCUSSION TITLE
Workflow for writing and updating notes

### DISCUSSION THREAD

**Discussion by joelgwebber** [brianpetro/obsidian-smart-connections#911] [Q&A]
Is anyone here aware of a good workflow for using the conversation window to read and _update_ notes?

I've been saving articles into Obsidian as a simple "read later" app, then using the conversation window to summarize and extract the high points for my notes. This works quite well, and it's not too much trouble to copy the output to a note.

But what I'd really love is the ability to have it [re-]summarize, organize, and update notes in place. E.g.,
- I have a bunch of poorly-organized bullet-point links to tools I'd like to try, spread out across daily notes. It would be really helpful to have it organize and categorize them into a single note for later follow-up.
- If I've created summaries of a number of articles, it would be helpful to be able to summarize the summaries and/or create directed extracts, and add them to a new or existing note.

All this is of course possible with the existing tools, but can be fairly manual and fiddly. I can reference a note reliably in a conversation, but it seems to fail if I try to reference a single block (using the anchor syntax). I find myself wanting a way to select a note or blocks therein, and prompt a model to rewrite them. Though perhaps there's a better way to get at these use-cases.

Sorry for the open-ended question, but I'm open to any and all ideas!

**Comment by brianpetro** [comment]
This is a great question. It's also something I'm constantly asking both myself and others.

While availability of tools is important, it is far from sufficient for making use of them. And the potential to use these tools recursively makes the possibilities near endless.

I suspect many current workflows are extensive/verbose, but sharing them will bring insights that will allow us as a community to compress parts and that enable new workflows that are more complex.

My hope is that we'll start sharing more workflows in a way that 1) brings more people up to speed about how they can empower themselves with the tools and 2) work together to refine our methods.

Thanks for posing the question. 🤞 I hope this starts some interesting discourse.

PS- some of the best discussions about workflows I've been a part of have been in our Community Lean Coffee meetings, here's the link (next one is tomorrow) https://lu.ma/calendar/cal-ZJtdnzAdURyouM7

🌴
